src/prasad/Comparecharinstrary.java

```diff
@@ -0,0 +1,42 @@
+package prasad;
+
+import java.util.Scanner;
+
+/*Accepts string array and then prints occurance of each char in string and repeats this for all strings in array*/
+
+public class Comparecharinstrary {
+	void compare(String[]words) {
+		for(int i=0;i<words.length;i++) {
+				String word = words[i];
+				System.out.println("");
+				for(int j=0;j<word.length();j++) {
```

Comment:
No require 3rd loop,
Refer this code
https://github.com/KrishnaKTechnocredits/JAVATechnoOct2020/pull/451/files